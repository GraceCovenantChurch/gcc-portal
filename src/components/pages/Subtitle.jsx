import React from "react";
import PropTypes from "prop-types";
import Typography from "@material-ui/core/Typography";

const Subtitle = (props) => {
  return (
    <Typography variant="h5" align="center" color="textSecondary" paragraph>
      {props.children}
    </Typography>
  );
};

Subtitle.propTypes = {
  children: PropTypes.node,
};

export default Subtitle;
