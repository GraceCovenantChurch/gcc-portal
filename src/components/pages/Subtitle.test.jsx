import React from "react";
import { shallow } from "enzyme";
import Subtitle from "./Subtitle";
import renderer from "react-test-renderer";

describe("Subtitle component", () => {
  let component, tree, wrapper;

  beforeEach(() => {
    component = renderer.create(<Subtitle>RANDOM TEXT</Subtitle>);
    tree = component.toJSON();

    wrapper = shallow(<Subtitle />);
  });

  it("UNIT: should be defined", () => {
    expect(Subtitle).toBeDefined();
  });

  it("SNAPSHOT: should render correctly", () => {
    expect(tree).toMatchSnapshot();
  });
});
