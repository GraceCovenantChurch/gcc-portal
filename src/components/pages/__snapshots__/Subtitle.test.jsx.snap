// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subtitle component SNAPSHOT: should render correctly 1`] = `
<p
  className="MuiTypography-root MuiTypography-h5 MuiTypography-colorTextSecondary MuiTypography-paragraph MuiTypography-alignCenter"
>
  RANDOM TEXT
</p>
`;
